model/ftn/w3src4md.ftn

```diff
@@ -155,26 +159,29 @@
 !
 !       !/S      Enable subroutine tracing.
 !       !/T      Enable test output.
+!       !/FLX5  Direct use of stress from atmoshperic model.
 !
 ! 10. Source code :
 !
 !/ ------------------------------------------------------------------- /
       USE W3ODATMD, ONLY: IAPROC
-      USE CONSTANTS, ONLY: TPIINV
+      USE CONSTANTS, ONLY: TPIINV, GRAV
       USE W3GDATMD, ONLY: NK, NTH, NSPEC, SIG, DTH, DDEN, WWNMEANP, &
                           WWNMEANPTAIL, FTE, FTF, SSTXFTF, SSTXFTWN,&
                           SSTXFTFTAIL, SSWELLF, ESIN, ECOS, AAIRCMIN, &
-                          AAIRGB
+                          AAIRGB, AALPHA, ZZWND
 !/S      USE W3SERVMD, ONLY: STRACE
 !/T      USE W3ODATMD, ONLY: NDST
 !/T      USE W3ODATMD, ONLY: NDST
 !
+!/FLX5      USE W3FLX5MD 
       IMPLICIT NONE
 !/
 !/ ------------------------------------------------------------------- /
 !/ Parameter list
 !/
       REAL, INTENT(IN)        :: A(NTH,NK), CG(NK), WN(NK), U, UDIR
+!/FLX5       REAL, INTENT(IN)        :: TAUA, TAUADIR, DAIR
```

Comment:
!/FLX5 should be removed here. The type is required for other FLXx options. It is quite relevant to the point @ukmo-ccbunney raised.

Reply:
!FLX5 was removed.

Reply:
!FLX5 needs to be added in this line after the addition of @ukmo-ccbunney of the switch in the subroutine as
` SUBROUTINE W3SPR4 (A, CG, WN, EMEAN, FMEAN, FMEAN1, WNMEAN,     &`
`                              AMAX, U, UDIR,                                    &`
`!/FLX5                            TAUA, TAUADIR, DAIR,                              &`
`                              USTAR, USDIR,                                     &`
`                              TAUWX, TAUWY, CD, Z0, CHARN, LLWS, FMEANWS, DLWMEAN)`
then when defining the parameter list this becomes:
`!/ Parameter list`
`!/`
`      REAL, INTENT(IN)        :: A(NTH,NK), CG(NK), WN(NK), U, UDIR`
`!/FLX5      REAL, INTENT(IN)        :: TAUA, TAUADIR, DAIR`
`      REAL, INTENT(IN)        :: TAUWX, TAUWY`

Reply:
well, if you do it, some tests will not pass, try the following test:
./bin/ run_test -b slurm -c hera.intel -S -T -s ST4_TSA -w work_ST4_TSA ../model ww3_ts1

Reply:
yes I see... I will wait for @ukmo-ccbunney to come back to me as we didn't have this issue when we were not specifying the new switch in order to avoid passing RHOAIR, TAUA and TUADIR to the source term subroutines all the time.